src/Components/Navbar.js

```diff
@@ -1,20 +1,28 @@
-import { Link } from "react-router-dom";
+import { NavLink } from "react-router-dom";
+import { useAuth } from "../dataProvider/AuthProvider";
 import { useCart } from "../dataProvider/CartProvider";
 import { SearchBar } from "./SearchBar";
+import { WelcomeNote } from "./WelcomeNote";
 
 export function Navbar() {
-  const {itemsInCart, wishList} = useCart();
+  const { itemsInCart, wishList, setShowSearchPage, showSearchPage } = useCart();
+  const { isLogin,email } = useAuth();
   return <div className="nav-bar">
     <img
         src="https://themeforshop.github.io/kala-pottery-demo/images/logo.png"
         alt="logo"
       />
-    <nav>
-      <Link to="/"> Home </Link>
-      <Link to="/products"> Products </Link>
-      <Link to="/cart">Cart <span>{itemsInCart.length}</span></Link>
-      <Link to="/wishlist">WishList <span>{wishList.length}</span></Link>
-      <SearchBar />
-    </nav>
+      <div className="btn">
+        <img src="https://www.flaticon.com/svg/vstatic/svg/3388/3388829.svg?token=exp=1619272488~hmac=31cd8170d45da88e513d33e2007c3b2c"  alt="hamburger-icon" className="ham"/>

```

Comment:
Hamburger icon is not visible. The navigation bar in mobile view stays even after clicking on a link.